tests_zemu/tests/3path.test.ts

```diff
@@ -0,0 +1,926 @@
+ *  (c) 2020 Zondax GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************* */
+
+import Zemu, {DEFAULT_START_OPTIONS, DeviceModel} from "@zondax/zemu";
+// @ts-ignore
+import {OasisApp} from "@zondax/ledger-oasis";
+
+const ed25519 = require("ed25519-supercop");
+const sha512 = require("js-sha512");
+
+const Resolve = require("path").resolve;
+const APP_PATH_S = Resolve("../app/output/app_s.elf");
+const APP_PATH_X = Resolve("../app/output/app_x.elf");
+
+const APP_SEED = "equip will roof matter pink blind book anxiety banner elbow sun young"
+
+const defaultOptions = {
+  ...DEFAULT_START_OPTIONS,
+  logging: true,
+  custom: `-s "${APP_SEED}"`
+};
+
+jest.setTimeout(60000)
+
+export const models: DeviceModel[] = [
+  {name: 'nanos', prefix: 'S', path: APP_PATH_S},
+  {name: 'nanox', prefix: 'X', path: APP_PATH_X}
+]
+
+describe('Standard-3Path', function () {
+  test.each(models)('can start and stop container', async function (m) {
+    const sim = new Zemu(m.path);
+    try {
+      await sim.start({...defaultOptions, model: m.name,});
+    } finally {
+      await sim.close();
+    }
+  });
+
+  test.each(models)('main menu', async function (m) {
+    const sim = new Zemu(m.path);
+    try {
+      await sim.start({...defaultOptions, model: m.name,});
+      await sim.compareSnapshotsAndAccept(".", `${m.prefix.toLowerCase()}-3path-mainmenu`, 3);
+    } finally {
+      await sim.close();
+    }
+  });
+
+  test.each(models)('get app version', async function (m) {
+    const sim = new Zemu(m.path);
+    try {
+      await sim.start({...defaultOptions, model: m.name,});
+      const app = new OasisApp(sim.getTransport());
+      const resp = await app.getVersion();
+
+      console.log(resp);
+
+      expect(resp.return_code).toEqual(0x9000);
+      expect(resp.error_message).toEqual("No errors");
+      expect(resp).toHaveProperty("test_mode");
+      expect(resp).toHaveProperty("major");
+      expect(resp).toHaveProperty("minor");
+      expect(resp).toHaveProperty("patch");
+    } finally {
+      await sim.close();
+    }
+  });
+
+  test.each(models)('get address', async function (m) {
+    const sim = new Zemu(m.path);
+    try {
+      await sim.start({...defaultOptions, model: m.name,});
+      const app = new OasisApp(sim.getTransport());
+
+      const path = [44, 474, 5];
```

Comment:
I think it would make the code clearer to use the string path (which shows which components are hardened) and use the account 0 which is [the first account in the second test vector in ADR 0008](https://github.com/oasisprotocol/oasis-core/blob/master/docs/adr/0008-standard-account-key-generation.md#test-vectors).
```suggestion
      const path = "m/44'/474'/0'";
```

---

The same applies to paths in other test cases.

Reply:
Added new set of tests!

Reply:
Thanks!